src/Python/Meshes/Decimation.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python
+
+import os.path
+
+import vtk
+
+
+def get_program_parameters():
+    import argparse
+    description = 'Decimate polydata.'
+    epilogue = '''
+    This is an example using vtkDecimatePro to decimate input polydata, if provided, or a sphere otherwise.
+    '''
+    parser = argparse.ArgumentParser(description=description, epilog=epilogue)
+    parser.add_argument('filename', nargs='?', default=None, help='Optional input filename e.g cow.g.')
+    parser.add_argument('reduction', nargs='?', type = float, default=None, help='Optional reduction percentage e.g 0.9')
+    args = parser.parse_args()
+    return args.filename, args.reduction
+
+
+def main():
+    filePath, reduction = get_program_parameters()
+    
+    # Define colors
+    colors = vtk.vtkNamedColors()
+    backFaceColor = colors.GetColor3d("gold")
+    inputActorColor = colors.GetColor3d("flesh")
+    decimatedActorColor = colors.GetColor3d("flesh")
+
+    if filePath and os.path.isfile(filePath):
+        inputPolyData = ReadPolyData(filePath)
+        if not inputPolyData:
+            inputPolyData = GetSpherePD()
+    else:
+        inputPolyData = GetSpherePD()
+
+    if not reduction:
+        reduction = .9 # 90% reduction
+
```

Comment:
The above two lines are not needed if the default is set to 0.9 in `get_program_parameters()`.